tests/test_parser.py

```diff
@@ -1133,3 +1139,101 @@ def test_ending_newlines(self):
                                                   wordnode('echo'),
                                                   wordnode('hello'),
                                                   wordnode('world')))
+
+    def test_case_simplest(self):
+      s = 'case ${1} in esac'
+      self.assertASTEquals(s,
+        compoundnode(s,
+          casenode(s,
+            reservedwordnode('case', 'case'),
+            wordnode('${1}', '${1}', [
+                parameternode('1', '${1}'),
+            ]),
+            reservedwordnode('in', 'in'),
+            reservedwordnode('esac', 'esac'),
+      )))
+
+    def test_case_claues(self):
```

Comment:
typo at clause

Reply:
Thank you for reviewing! I fixed the typo.